lang/cs/Org.Apache.REEF.Client/Local/LocalClient.cs

```diff
@@ -132,16 +134,23 @@ public IDriverHttpEndpoint SubmitAndGetDriverUrl(IJobSubmission jobSubmission)
                 ));
 
             var fileName = Path.Combine(driverFolder, _fileNames.DriverHttpEndpoint);
-            HttpClientHelper helper = new HttpClientHelper();
-            _driverUrl = helper.GetDriverUrlForLocalRuntime(fileName);
+            JobSubmissionResultImpl result = new JobSubmissionResultImpl(this, fileName, false);
 
-            Logger.Log(Level.Info, "Submitted the Driver for execution. Returned driverUrl is: " + _driverUrl);
-            return helper;
+            Logger.Log(Level.Info, string.Format(CultureInfo.CurrentCulture,
+                "Submitted the Driver for execution. Returned driverUrl is: {0}.", result.DriverUrl));
+            return result;
         }
 
-        public string DriverUrl
+        /// <summary>
+        /// Return current Job status
+        /// </summary>
+        /// <returns></returns>
+        /// //TODO: REEF-889
+        [Unstable("0.14", "Working in progress for rest API status returned")]
+        public async Task<FinalState> GetJobFinalStatus(string appId)
         {
-            get { return _driverUrl; }
+            await Task.Delay(1000); //1 seconds delay
```

Comment:
This is just a place holder as we have to implement the interface which match the async syntax. It is TODO. I changed the delay to 0.